### ISSUE TITLE
Version 0.0.0 not accepted for requirement (any)

### ISSUE THREAD

**Issue by dradetsky** [emacs-eldev/eldev#107]
**Describe the bug**

I created a package and since eldev insisted it have a version, I added

```
;; Version: 0.0.0
```

I then went to run my tests with `eldev test` which triggered

```
Dependency ‘ignore-mouse’ version (any) is required, but at most 0.0.0 is available
```

And, uhh, that's not what "any" means, buddy.

Anyhow, I was able to fix this by setting the version to 0.0.1. So this is not hard to work around, but also maybe a sign that something is wrong. And in any case, if this is a requirement eldev wants to enforce for whatever reason, it's not clear that it actually attempting to enforce that requirement instead of just suffering from a bug. And also, it's not a requirement I care for.

**Steps to reproduce**

Create a file like
```
;;; replicate.el --- replicate  -*- lexical-binding: t -*-

;; Version: 0.0.0

;;; replicate.el ends here
```

and an Eldev like 

```
(eldev-use-package-archive 'melpa)
```

And run `eldev test` in that directory.

**Expected behavior**

Complaint about inability to find any test files or something like that.

**Actual behavior**

```
Dependency ‘replicate’ version (any) is required, but at most 0.0.0 is available
```

**Environment**

- Eldev: 1.10.3
- Emacs: 29.4
- OS: Arch Linux

**Comment by doublep** [comment]
> eldev insisted it have a version

Eldev requires it's a package from Emacs point of view. It's rather Emacs [package system] that insists on some version.

The rest is indeed a bug. I committed a fix; it will be released later, with 1.11, as it is a minor issue with a trivial workaround.

**Comment by doublep** [comment]
Fixed in 1.11.
